frontend/src/app/Settings/ManageOrganizationContainer.tsx

```diff
@@ -90,7 +64,7 @@ const ManageOrganizationContainer: any = () => {
 
   return (
     <ManageOrganization
-      organization={data.organization}
+      organization={data.whoami.organization as OrganizationForm}
```

Comment:
Similar to comment above. This mapping should map to the `Organization` type defined in `../../generated/graphql` as is basically receiving the endpoint response.